docs/usage.rst

```diff
@@ -27,9 +27,9 @@ Install all Python requirements.
    $ python -m venv .venv
 
 .. code-block:: console
-   
+
    $ source .venv/bin/activate
-   
+
 .. code-block:: console
 
    (.venv) $ pip install -r requirements/requirements-dev.txt -r requirements/requirements.detector.txt -r requirements/requirements.logcollector.txt -r requirements/requirements.prefilter.txt -r requirements/requirements.inspector.txt
```

Comment:
We're also missing the `requirements.logserver.txt` here.

Reply:
The `requirements-logserver.txt` has been added by #32 
I prefer to fix it in another PR.